lib/manager/helmv3/readme.md

```diff
@@ -3,3 +3,51 @@ Renovate supports updating Helm Chart references within `requirements.yaml` (Hel
 If your Helm charts make use of repository Aliases then you will need to configure an `aliases` object in your config to tell Renovate where to look for them.
 
 If you need to change the versioning format, read the [versioning](https://docs.renovatebot.com/modules/versioning/) documentation to learn more.
+
+### Private repositories and registries
+
+To use private sources of Helm charts, HostRules have to be defined.
```

Comment:
```suggestion
To use private sources of Helm charts, you must set the password and username you use to authenticate to the private source.
For this you use a custom `hostRules` array.
```

How about (something like) this?
I think this also explains _why_ you need the custom `hostRules`. 😉